tests/test_invalid_versioned_playlists.py

```diff
@@ -0,0 +1,24 @@
+# coding: utf-8
+# Copyright 2014 Globo.com Player authors. All rights reserved.
+# Use of this source code is governed by a MIT License
+# license that can be found in the LICENSE file.
+
+import invalid_versioned_playlists
+import pytest
+
+import m3u8
+
+
+@pytest.mark.xfail
+def test_should_fail_if_iv_in_EXT_X_KEY_and_version_less_than_2():
+    m3u8.parse(invalid_versioned_playlists.M3U8_RULE_IV)
```

Comment:
nit: maybe we should assert the exception or at least partially the error message/help

```python
def test_raises():
    with pytest.raises(Exception) as exc_info:   
        raise Exception('some info')
    # these asserts are identical; you can use either one   
    assert exc_info.value.args[0] == 'some info'
    assert str(exc_info.value) == 'some info'
```

ref https://stackoverflow.com/a/23514853

Reply:
ref https://docs.pytest.org/en/7.1.x/how-to/assert.html

Reply:
ref https://pytest-with-eric.com/introduction/pytest-assert-exception/